db/data_migration/20150514105208_add_eu_logo_url_to_eafrd_editions.rb

```diff
@@ -0,0 +1,24 @@
+slugs = [
+  "countryside-stewardship-facilitation-funding",
+  "countryside-stewardship-water-capital-grants-2015-capital-items",
+  "countryside-stewardship-water-capital-grants-catchment-sensitive-farming",
+  "countryside-stewardship-woodland-capital-grants-2015",
+  "countryside-productivity-scheme",
+  "rural-development-programme-for-england-leader-funding"
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/media/55547b94ed915d15d8000057/european-agricultural-fund-for-rural-development.gif"
```

Comment:
Makes sense. It does look like a brittle URL though. Is that the case? i.e. if the logo is changed or updated, does the URL remain the same or will these editions need to be updated with the new URL?